dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/CCLicenseFeatureRestIT.java

```diff
@@ -0,0 +1,266 @@
+        Authorization authAdminCCLicense = new Authorization(admin, ccLicenseFeature, itemRest);
+
+        // access the authorization for the admin user
+        String adminToken = getAuthToken(admin.getEmail(), password);
+        getClient(adminToken).perform(get("/api/authz/authorizations/" + authAdminCCLicense.getID()))
+                    .andExpect(status().isOk())
+                    .andExpect(jsonPath("$",
+                            Matchers.is(AuthorizationMatcher.matchAuthorization(authAdminCCLicense))));
+
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/objectAndFeature")
+                .param("uri", itemUri)
+                .param("eperson", admin.getID().toString())
+                .param("feature", ccLicenseFeature.getName()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$",
+                    Matchers.is(AuthorizationMatcher.matchAuthorization(authAdminCCLicense))));
+    }
+
+    @Test
+    public void checkAuthorizationAsCommunityAdminTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Community com = CommunityBuilder.createCommunity(context).withName("A community").withAdminGroup(eperson)
+                .build();
+        Collection col = CollectionBuilder.createCollection(context, com).withName("A collection").build();
+        Item item = ItemBuilder.createItem(context, col).withTitle("Item to withdraw").build();
+        context.restoreAuthSystemState();
+
+        ItemRest itemRest = converterService.toRest(item, converterService.getProjection(DefaultProjection.NAME));
+        String itemUri = utils.linkToSingleResource(itemRest, "self").getHref();
+        Authorization authAdminCCLicense = new Authorization(eperson, ccLicenseFeature, itemRest);
+
+        // access the authorization for the community admin user
+        String comAdminToken = getAuthToken(eperson.getEmail(), password);
+        getClient(comAdminToken).perform(get("/api/authz/authorizations/" + authAdminCCLicense.getID()))
+                    .andExpect(status().isOk())
+                    .andExpect(jsonPath("$",
+                            Matchers.is(AuthorizationMatcher.matchAuthorization(authAdminCCLicense))));
+
+        getClient(comAdminToken).perform(get("/api/authz/authorizations/search/objectAndFeature")
+                .param("uri", itemUri)
+                .param("eperson", eperson.getID().toString())
+                .param("feature", ccLicenseFeature.getName()))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$",
+                    Matchers.is(AuthorizationMatcher.matchAuthorization(authAdminCCLicense))));
+
+        // verify that the property core.authorization.collection-admin.item-admin.cc-license = false is respected
+        // the community admins should be still authorized
+        configurationService.setProperty("core.authorization.item-admin.cc-license", false);
```

Comment:
I assume this should not be disabled here?

Reply:
No it is intentional as stated in the comment. I'm testing that in the case the feature is reserved to the community administrator he can still access the feature, so I "restrict" the feature disabling access for item and collection admins

Reply:
But as far as I can see in the test, you're only testing for a collection admin and not an item admin. I would have only expected the change to `core.authorization.collection-admin.item-admin.cc-license`

Reply:
the delegation of authorization require that you disable all the lower level to work. No changes on this, it works in this way also in the current version. If we turnoff "only for collection admin" without turnoff for item admins the collections admin will be still able to do as they are ALSO item admins.
Test specific for collection admins and item admins are in the dedicated test methods
`checkAuthorizationAsCollectionAdminTest`
`checkAuthorizationAsItemAdminTest`